dapps/marketplace/src/components/Reviews.js

```diff
@@ -173,26 +182,41 @@ export default class Reviews extends Component {
 require('react-styl')(`
   .reviews
     margin-bottom: 2rem
+    h3
+      font-family: Poppins
+      font-size: 1.5rem
+      font-weight: 500
+      font-style: normal
+      font-stretch: normal
+      line-height: 1.42
+      letter-spacing: normal
+      color: var(--dark)
     .review
       .user-info
         display: flex
-        width: 100%;
+        width: 100%
         align-items: flex-start
         .avatar
           margin-right: 1rem
         .user
           min-width: 0
           flex: 1
+          margin-top: 0.6rem
           .top
             flex: 1
             min-width: 0
             display: flex
             flex-direction: row
             align-items: baseline
             .name
-              font-size: 18px
+              font-family: Lato
+              font-size: 1.1rem
               font-weight: bold
-              color: var(--black)
+              font-style: normal
```

Comment:
From my experience in most cases these are not needed: 
```
              font-style: normal
              font-stretch: normal
              line-height: normal
              letter-spacing: normal
```

Reply:
`line-height` probably affects a few places. I'll remove the other things if unnecessary.